LICENSE

```diff
@@ -173,30 +173,3 @@
       defend, and hold each Contributor harmless for any liability
       incurred by, or claims asserted against, such Contributor by reason
       of your accepting any such warranty or additional liability.
-
-   END OF TERMS AND CONDITIONS
```

Comment:
fwiw, I kept the `END OF TERMS AND CONDITIONS` in the ledger license (but did definitely delete the appendix).

Reply:
I put the line back.  Thanks!

Reply:
I put the line back.  Thanks!